lib/wallaby/query/error_message.ex

```diff
@@ -174,6 +174,15 @@ defmodule Wallaby.Query.ErrorMessage do
   end
   def condition(_), do: nil
 
+  @spec visibility_and_selection(Query.t) :: String.t
+  defp visibility_and_selection(query) do
```

Comment:
This is how we're managing the combined error message `"... 1 visible, selected element, but ..."`. I'm happy to change this if there's a better way or to remove it if we don't want to include `selected/unselected` in the error message.